spec.bs

```diff
@@ -135,12 +136,32 @@ interface AppHistory : EventTarget {
   readonly attribute boolean canGoBack;
   readonly attribute boolean canGoForward;
 
+  Promise<undefined> navigate(USVString url, optional AppHistoryNavigateOptions options = {});
+  Promise<undefined> navigate(optional AppHistoryNavigateOptions options = {});
```

Comment:
Replace is not actually consulted for this overload. Perhaps we should use a fresh dictionary? This is actually observable in that the getter for `replace` will be called, whereas if we used a fresh dictionary it would not be called.

Reply:
Web IDL requires that trailing dictionaries be optional, even if logically they are not optional. See https://github.com/heycam/webidl/issues/903.

Probably we should include a comment since this is confusing to people.